src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -342,6 +342,9 @@
 helps['network dns zone create'] = """
     type: command
     short-summary: Creates or updates DNS zone properties. Does not modify DNS records within the zone.
+    parameters:
+        - name: --if-none-match
```

Comment:
But at the same time, the service team must have some reason for exposing it, so we generally try to expose whatever functionality the SDK exposes.